frontend/src/AddTask.js

```diff
@@ -8,6 +8,7 @@ import { useInsertTask } from "./api/api.hooks";
 
 const schema = z.object({
   title: z.string(),
+  effort: z.any(),
```

Comment:
Can we use the actual type here rather than `any`? It makes it easier the validation and handling process.